manifests/w/Wondershare/Filmora/10.5.7.4/Wondershare.Filmora.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2021/Aug/19
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Wondershare.Filmora
+PackageVersion: 10.5.7.4
+PackageLocale: en-US
+Publisher: Wondershare Software
+PublisherUrl: "https://filmora.wondershare.com/"
+PublisherSupportUrl: https://support.wondershare.com
+PrivacyUrl: "https://www.wondershare.com/privacy.html"
+PackageName: "Wondershare Filmora"

```

Comment:
```suggestion
PackageName: "Wondershare Filmora X"
```
I think this is supposed to be `Wondershare Filmora X`?
![image](https://user-images.githubusercontent.com/11600822/130041687-99228db0-29e5-4ffb-9073-58415280f86d.png)